Extension/i18n/chs/walkthrough/devcommandprompt/open-developer-command-prompt.md.i18n.json

```diff
@@ -4,8 +4,6 @@
  *--------------------------------------------------------------------------------------------*/
 // Do not edit this file. It is machine generated.
 {
-	"walkthrough.windows.title.open.dev.command.prompt": "使用 {0} 重新启动",
```

Comment:
I had to rename the properties otherwise `translations-generate` crashes when it doesn't find an html element to match the {0}, {1} items.  The new strings don't have a substitution.  Since the properties are renamed, these ones are dead code.